hooks/pre/30-upgrade.rb

```diff
@@ -1,3 +1,13 @@
+def stop_services
+  Kafo::Helpers.execute('katello-service stop')
+end
+
+def start_mongo
+  status = Kafo::Helpers.execute('service-wait mongod status')
+  Kafo::Helpers.execute('service-wait mongod start') unless status
```

Comment:
I've not seen that problem myself, does service-wait just need a longer timeout or something? We can't continue with the pulp migration unless mongo is confirmed started.